launch/global_mapper_nodelet.launch

```diff
@@ -15,11 +15,12 @@
 
       <!-- remaps -->
       <remap from="cloud_topic" to="/flight/xtion_depth/points" />
-      <remap from="~map_topic" to="~global_map" />
+      <remap from="~pixel_map_topic" to="~pixel_map" />
+      <remap from="~voxel_map_topic" to="~voxel_map" />
```

Comment:
I think I would agree with that guidance, but since the remapped topic names aren't args here, it doesn't seem like you are actually able to remap them when you call this launch file. Is there a reason not to do something like this:
```xml
<arg name="pixel_map" default="pixel_map />
  ...
  <remap from ="~pixel_map" to="~($arg pixel_map)" />
```